python/ray/autoscaler/gcp/config.py

```diff
@@ -75,6 +72,7 @@ def wait_for_compute_global_operation(project_name, operation):
 
 def key_pair_name(i, region, project_id, ssh_user):
     """Returns the ith default gcp_key_pair_name."""
+    # TODO: Figure out why this doesn't actually use i
```

Comment:
Good point, maybe @hartikainen knows what should happen here?

Reply:
Isn't i used below on line 76?

Reply:
Oh... maybe should just add another `_{}`. I'm surprised that doesn't raise an error.

Reply:
Seems like my bug. I'm pretty sure that, as Eric said, this should have another `_{}`.